@navikt/core/react/src/form/combobox/SelectedOptions/selectedOptionsContext.tsx

```diff
@@ -101,6 +102,14 @@ const SelectedOptionsProvider = ({
     [customOptions, onToggleSelected, removeCustomOption],
   );
 
+  useEffect(() => {
+    const nextIsLimitReached =
+      (!!maxSelected?.limit && selectedOptions.length >= maxSelected.limit) ||
+      !isMultiSelect;
```

Comment:
Caret is hidden when deselecting on single select. Maybe `(!isMultiSelect && selectedOptions.length > 0)`

Reply:
hadn't thought about that case! nice 🙌